cpu/cortexm_common/mpu.c

```diff
@@ -36,8 +36,11 @@ int mpu_enable(void) {
 #if __MPU_PRESENT
     MPU->CTRL |= MPU_CTRL_PRIVDEFENA_Msk | MPU_CTRL_ENABLE_Msk;
 
-    /* Enable the memory fault exception */
+#if !defined(CPU_ARCH_CORTEX_M0PLUS)
```

Comment:
I think it is simpler to just protect this with `#ifdef SCB_SHCSR_MEMFAULTENA_Msk`